vpp/vaapipostprocess_base.cpp

```diff
@@ -70,6 +65,28 @@ void VaapiPostProcessBase::cleanupVA()
     m_display.reset();
 }
 
+YamiStatus VaapiPostProcessBase::queryVideoProcFilterCaps(
+    VAProcFilterType filterType, void* filterCaps, uint32_t* numFilterCaps)
+{
+    if (!filterCaps)
```

Comment:
no, current no user case for filterCaps == NULL && numFilterCaps != NULL, all caller will provide filterCaps. I do not want this function too complicate